package.json

```diff
@@ -68,7 +68,7 @@
   "runtimeDependencies": [
     {
       "description": "Mono Runtime (Linux / x86)",
-      "url": "https://omnisharpdownload.blob.core.windows.net/ext/mono.linux-x86-5.2.0.104.zip",
+      "url": "https://omnisharpdownload.azureedge.net/ext/mono.linux-x86-5.2.0.104.zip",
       "installPath": "./bin",
```

Comment:
You probably also want a `fallbackUrl` with the original URL set. We found some folks couldn't connect to the CDN.